axelrod/tests/unit/test_classification.py

```diff
@@ -38,7 +38,7 @@ def test_multiple_instances(self):
         self.assertNotEqual(P1.classifier, P2.classifier)
 
         P1 = axelrod.GoByMajority(5)
-        P2 = axelrod.StochasticWSLS(10)
+        P2 = axelrod.StochasticWSLS(.1)
```

Comment:
This was failing because `1 - 10 < 0 < 1` and was caught in the ValueError added